src/App.tsx

```diff
@@ -1,23 +1,44 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
 import { BrukerDetaljerPage } from './component/page/bruker-detaljer/BrukerDetaljerPage';
 import { TiltaksoversiktPage } from './component/page/tiltaksoversikt/TiltaksoversiktPage';
 import StoreProvider from './store/store-provider';
+import { LoginPage } from './component/page/login/LoginPage';
+import { Spinner } from './component/felles/spinner/Spinner';
+import { checkIsAuthenticated } from './api';
 
 export const App = () => {
+	const [isAuthenticated, setIsAuthenticated] = useState<boolean>(false);
+	const [isLoading, setIsLoading] = useState<boolean>(true);
+
+	useEffect(() => {
+		checkIsAuthenticated()
+			.then((res) => setIsAuthenticated(res.data.isAuthenticated))
```

Comment:
Kan disse være navngitt på norsk uten at det blir rart. SetErAutentisert høres ok ut for meg

Reply:
Det kan det. Tror det tar en god stund før jeg vender meg til å skrive kode på norsk.
Men fikk vi egentlig konkludert at vi ønsket å skrive all koden på norsk? Trodde vi mer lander på engelsk kode men norsk ord for fagting og annet som er vanskelig på engelsk